lib/gstreamer/msdk/encoder.py

```diff
@@ -142,6 +142,13 @@ def before(self):
     super().before()
     os.environ["GST_MSDK_DRM_DEVICE"] = get_media().render_device
 
+    # WA: Fix the gst-discoverer-1.0.exe report Missing plugins problem on Windows OS
+    if platform.info()['os'] == 'wsl':
+      self.__rank_before = os.environ.get("GST_PLUGIN_FEATURE_RANK", None)
+      ranks = [] if self.__rank_before is None else self.__rank_before.split(',')
+      ranks += [f"{e}:MAX" for e in get_elements("msdk")]
+      os.environ["GST_PLUGIN_FEATURE_RANK"] = ','.join(ranks)
+
```

Comment:
Please restore GST_PLUGIN_FEATURE_RANK setting after the test is finished (i.e. `def after(self)...`).  See transcoder test class for example.